pyapp/cookiecutter.json

```diff
@@ -15,6 +15,7 @@
     "postgres": ["no", "yes"],
     "docker": ["no", "yes"],
     "__postgres_port": "{{ random_port_number() }}",
+    "__postgres_version": "15.3-alpine",
```

Comment:
Whoops, the `__postgres_version` option wasn't working because it was missing from these `cookiecutter.json` files